config/initializers/bus_location_simulator.rb

```diff
@@ -0,0 +1,38 @@
+if ENV['ENABLE_BUS_LOCATION_SIMULATOR'] == 'true'
+  # Changes bus location with every 5 sec to simulate real-time tracking
+  require 'rufus-scheduler'
+
+  def convert_to_radians(degrees)
+    degrees * Math::PI / 180
+  end
+
+  def convert_to_degrees(radians)
+    radians * 180 / Math::PI
+  end
+
+  #These calculations simulate the bus driving around in a circle
+  earth_radius = 3437.670013352 # mi
+  radius_of_circle = 1 # mi
+  scheduler = Rufus::Scheduler.new
+  distance = radius_of_circle / earth_radius
+  iteration = 0
+  scheduler.every '5s' do
+    Bus.all.each do |bus|
+      degree = convert_to_radians iteration
+      bus_location = bus.bus_locations.first
+      bus_heading = bus_location.heading
+
+      lat = convert_to_radians bus_location.latitude
+      long = convert_to_radians bus_location.longitude
+
+      new_latitude = convert_to_degrees(Math.asin(Math.sin(lat) * Math.cos(distance) + Math.cos(lat) * Math.sin(distance) * Math.cos(degree)))
+      new_longitude = convert_to_degrees(long + Math.atan2(Math.sin(degree) * Math.sin(distance) * Math.cos(lat), Math.cos(distance) - Math.sin(lat) * Math.sin(new_latitude)))
+
+      BusLocation.create(bus_id: bus.id, latitude: new_latitude,
+                         heading: bus_heading, longitude: new_longitude, recorded_at: Time.now
+      )
+    end
+    iteration += 1
+    iteration = 0 if iteration >= 360 # reset circle
+  end
+end
```

Comment:
Nice! Can we move this implementation to it's own class? 
`class BusLocationSimulator` 

Then we can use the initializer to call it.